packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -697,6 +703,15 @@ contract OPContractsManager_AddGameType_Test is Test {
         IOPContractsManager.AddGameInput[] memory inputs = new IOPContractsManager.AddGameInput[](1);
         inputs[0] = input;
 
+        uint256 l2ChainId = IFaultDisputeGame(
+            address(IDisputeGameFactory(input.systemConfig.disputeGameFactory()).gameImpls(GameType.wrap(1)))
+        ).l2ChainId();
+
+        // Expect the GameTypeAdded event to be emitted.
+        vm.expectEmit(true, true, false, false, address(this));
```

Comment:
we should have a semgrep rule for this, all true is a safer default that way if we change the event sig everything is tested by default 
```suggestion
        vm.expectEmit(address(this));
```

Reply:
doesn't it seem a bit much to have to precompute the addresses the game and weth will be deployed at?

Reply:
They are not `indexed`, so true, true, true, false would be fine with me. Or maybe we just don't need them in the event.

Reply:
it's "simple" but super ugly because the initcode includes either 8 or 10 constructor args depending on the game type. Thanks for not blocking on this, resolving.